python/src/opendp/_extrinsics/_make_private_selection/__init__.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional
+
+import opendp.prelude as dp
+from opendp._lib import import_optional_dependency
+from opendp.mod import Measurement
+
+
+def make_private_selection_threshold(meas: Measurement,
+                                     threshold: float,
+                                     stop_probability: float,
+                                     epsilon_selection: float,
+                                     steps: Optional[int] = None):
+
+    """Measurement for private selection with known threshold.
+
+    This executes an ε-DP mechanism M that returns a tuple (q, x) repeatedly
+    until the score q is at least T, a predefined threshold.
+    Only the last output of M is released.
+
+    Algorithm 1: https://arxiv.org/pdf/1811.07971.pdf#page=7
```

Comment:
Make this an RST link with the title as the clickable text?

```suggestion
    Algorithm 1 in `Private selection from private candidates <https://arxiv.org/pdf/1811.07971.pdf#page=7>`_
```